canonical-web-and-design/charmhub.io
# Implement static features list and platform filtering

## Done
Implement static features list and platform filtering

**Needs the featured charm data**

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://localhost:8045
- Check the set of static cards appear instantly and the count updates when the rest load
- Change the platform and see that the featured charms are replaced with a filtered list quickly
- See that the list count is correct
- Change the platform to All
- Check you see them all and the count is rendered correctly

## Issue / Card

Fixes https://github.com/canonical-web-and-design/charmhub.io/issues/584
